freud/cluster.pyx

```diff
@@ -180,12 +180,13 @@ cdef class ClusterProperties(_Compute):
     def __dealloc__(self):
         del self.thisptr
 
-    def compute(self, system, cluster_idx):
+    def compute(self, system, cluster_idx, masses=None):
         r"""Compute properties of the point clusters.
-        Loops over all points in the given array and determines the center of
-        mass of the cluster as well as the gyration tensor. After calling
-        this method, these properties can be accessed with the
-        :code:`centers` and :code:`gyrations` attributes.
+        Loops over all points in the given array and determines the geometric
+        centers, centers of mass of the, moment of inertia and gyration tensors,
```

Comment:
I think this is missing something. Either that or I'm finding it very difficult to read properly.

Reply:
Ah yes, definitely a typo on my part, thanks!